### PR TITLE
Vim network session microsoft sysmon

### PR DESCRIPTION
**Pull request by goosvorbook**
Required items, please complete
   
   Change(s):
Added vim parser for Sysmon Event ID 3 (both Event and WindowsEvent)

### CODE REVIEW

File: Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoftSysmon.yaml [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+         Image = tostring(EventData.Image),
+         User = tostring(EventData.User),
+         Protocol = tostring(EventData.Protocol),
+         Initiated = tobool(EventData.Initiated), // Initiated indicates the process initiated a connection (meaning outbound)
+         SourceIsIpv6 = tobool(EventData.SourceIsIpv6),
+         SourcePort = toint(EventData.SourcePort),
+         SourcePortName = tostring(EventData.SourcePortName),
+         DestinationIsIpv6 = tobool(EventData.DestinationIsIpv6),
+         DestinationPort = toint(EventData.DestinationPort),
+         DestinationPortName = tostring(EventData.DestinationPortName)
```
Comment (devikamehra) [inline Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoftSysmon.yaml:189]: This value was extracted but never used.
Similarly in WindowsEvents also.

**Comment by v-vdixit** [comment]
Hi @goosvorbook please check the comments and make required changes, thanks

**Comment by v-prasadboke** [comment]
Hello @goosvorbook any updates on the above

**Comment by v-prasadboke** [comment]
Hello @goosvorbook please provide your update

**Comment by goosvorbook** [comment]
For now, consider this PR as a draft.

**Comment by v-prasadboke** [comment]
ok @goosvorbook please mark it as draft

**Comment by v-rbajaj** [comment]
Hi @goosvorbook, We wanted to check on the status of PR https://github.com/Azure/Azure-Sentinel/pull/7136  . PR is pending for more than expected days. Please let us know if you need any assistance to review this PR. Per our standard operating procedures if no response is received in the next 7 business days, we will close this PR. Thank you for your cooperation

**Comment by v-atulyadav** [comment]
Since we have not received a response in the last 7 days, we are closing your PR #7136 per our standard operating procedures. If you still need support for this issue, you can re-open the PR at any time. 
If you do re-open, we simply request that you ensure the PR has response to the last request. Thank you for your cooperation

**Pull request closed without merging**